code/modules/events/spacevine.dm

```diff
@@ -488,18 +488,15 @@
 	var/force = W.force
 
 	if(istype(W, /obj/item/weapon/scythe))
-		force = force * 4
-		for(var/obj/effect/spacevine/B in orange(1,src))
-			B.health = health - force
-			if(B.health < 1)
-				wither()
-
-		health = health - force
-
-		if(health < 1)
-			wither()
-
-		return
+		var/obj/item/weapon/scythe/S = W
+		if(S.extend)	//so folded telescythes won't get damage boosts / insta-clears (they instead will instead be treated like non-scythes)
+			force = force * 4
+			for(var/obj/effect/spacevine/B in range(1,src))

```

Comment:
I changed this to `range()` instead of `orange()` since we still affect the center vine as well as the surroundings. It just cleans up and condenses the code since we aren't treating the center vine differently than its neighbors.